public/manifest.json

```diff
@@ -22,4 +22,4 @@
   "display": "standalone",
   "theme_color": "#000000",
   "background_color": "#ffffff"
-}
+}
```

Comment:
Not sure this change is required?

Reply:
oops sorry, this was autoformatting

Reply:
resolved, my autoformatter removed the newline at the end of the file for some reason.